### ISSUE TITLE
Deploy Spark to staging

### ISSUE THREAD

**Issue by ripper234** [Midburn/spark#31]
Let's have the CI auto-deploy to our staging server.
Devops already provisioned a server for us.

We need to:

- [ ] Write an npm build target deploystaging
- [ ] Hook it to Travis
- [ ] Connect staging.camps.midburn.org to the server
- [ ] Update the readme with this address

**Comment by OriHoch** [comment]
I'm working on it

**Comment by OriHoch** [comment]
related issue: #34

**Comment by OriHoch** [comment]
some of this work was done in #45

**Comment by OriHoch** [comment]
I started working on it, but it's on hold now, waiting for:
1. merge of PR #45 
2. more details regarding how app will run on an actual server (this conflicts with work on Docker which someone else is doing)
3. credentials to access amazon to configure the CI/CD stuff

once those 3 points are resolved I could move on to add more features to CI/CD..

**Comment by OriHoch** [comment]
got the staging server, I'll continue some work on it

**Comment by OriHoch** [comment]
after #45 is merged we have automated deploy on every travis successful build to master

**Comment by eyalhag** [comment]
@OriHoch very cool, what else needs to be done in order to mark this task as finished?

**Comment by OriHoch** [comment]
original issue description contains some tasks which I don't fully understand what they mean.. 

The only thing I know is missing is DNS setting

**Comment by OriHoch** [comment]
some work is still needed

**Comment by amotenko** [comment]
what's missing?

**Comment by OriHoch** [comment]
we have some basic implementation but issue description contains some points which are not done yet IMO

I'm just not sure why you closed it.. if you think everything is done then feel free to close, just let us know..

**Comment by ripper234** [comment]
I think it can be closed, npm target isn't relevant since it auto-deploys, I think everything else is done, no?

**Comment by OriHoch** [comment]
I vaguely remember we discussed more stuff but if you can't think of
anything else then let's close it !

**Comment by amotenko** [comment]
worst case guys, we'll open a new one :metal:
